lock-api/src/main/java/com/palantir/lock/v2/TimelockService.java

```diff
@@ -78,4 +81,14 @@ default void tryUnlock(Set<LockToken> tokens) {
 
     long currentTimeMillis();
 
+    // todo(gmaretic): implement
+    default StartTransactionWithWatchesResponse startTransactionWithWatches() {
+        return StartTransactionWithWatchesResponse.of(
+                startIdentifiedAtlasDbTransaction(), LockWatchState.of(ImmutableMap.of()));
+    }
+
+    // todo(gmaretic): implement
+    default TimestampedLockResponse acquireLocksForWrites(LockRequest lockRequest) {
+        return TimestampedLockResponse.of(getFreshTimestamp(), lock(lockRequest));
```

Comment:
This is 1.5 RPCs, which is more than on the normal path

Reply:
Fair enough, fixed